### PR TITLE
Expose --publish option to Sigrid CI client script.

### PR DESCRIPTION
**Pull request by dennis-sig**

### CODE REVIEW

File: sigridci/sigridci.py [hunk 1]
```diff
@@ -271,23 +269,31 @@ def generate(self, feedback, args):
...
                 "(" + self.formatRating(feedback["overallRatings"], metric) + ")".ljust(21) + \
                 self.formatRating(feedback["newCodeRatings"], metric), feedback["newCodeRatings"].get(metric))
                 
+    def printMetric(self, feedback, metric):
+        print("")
+        print(metric.replace("_PROP", "").title().replace("_", " "))
+        
+        refactoringCandidates = self.getRefactoringCandidates(feedback, metric)
+        if len(refactoringCandidates) == 0:
+            print("    None")
```
Comment (bruno-oliveira) [inline sigridci/sigridci.py:292]: I assume these extra whitespaces are here for visual formatting only?

File: sigridci/sigridci-feedback-template.html [hunk 2]
```diff
@@ -280,7 +280,7 @@ <h3>Baseline (@@@BASELINE_DATE)</h3>
                             <span class="separator">@@@MAINTAINABILITY_STARS_OVERALL</span>
                         </div>
                         <div>
-                            <h3>New code quality</h3>
+                            <h3>New/changed code</h3>
```
Comment (bruno-oliveira) [inline sigridci/sigridci-feedback-template.html:283]: Maybe we can say here: `New/changed code quality` to be consistent with how it was described before?

File: sigridci/sigridci.py (hunk 1 above)
Comment (dennis-sig) [reply sigridci/sigridci.py:292]: You mean the extra whitespace in front of the "none" ?

File: sigridci/sigridci-feedback-template.html (hunk 2 above)
Comment (dennis-sig) [reply sigridci/sigridci-feedback-template.html:283]: You're right, let's make terminology consistent.

File: sigridci/sigridci.py (hunk 1 above)
Comment (bruno-oliveira) [reply sigridci/sigridci.py:292]: yes exactly

Comment (dennis-sig) [reply sigridci/sigridci.py:291]: It's for readability in the ASCII art output, e.g.:

```
Duplication
    None

Unit Size
    None
```

**Pull request merged**